spec/text_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'text'
+require 'time'
+
+describe Text do
+
+  it 'sends a text' do
+    fake_message = {
+      from: '123',
+      to: '456',
+      body: 'testing my message'
+    }
+
+    expect(subject).to receive(:send_text).with(fake_message)
+    subject.send_text(fake_message)
+  end
+
+  it 'has a delivery time of an hour' do
+    allow(Time).to receive(:now).and_return(Time.parse("19:00"))
```

Comment:
This one is better -> Stubbing a class/object that is not the subject. 
In this case, you can achieve similar results using a gem called `timecop`